Doc/make.bat

```diff
@@ -13,7 +13,7 @@ if not defined SPHINXBUILD (
     %PYTHON% -c "import sphinx" > nul 2> nul
     if errorlevel 1 (
         echo Installing sphinx with %PYTHON%
-        %PYTHON% -m pip install sphinx==2.2.0
+        %PYTHON% -m pip install -r requirements.txt
```

Comment:
The 3.7 branch does not have a requirements.txt for the Doc venv.

Reply:
Hah, I should've been checking that as I backported. Looks like it's a different fix for this version (and 3.8)